### ISSUE TITLE
Long keywords lists in search take too much place

### ISSUE THREAD

**Issue by rumkin** [npm/www#386]
## Link to the page

https://www.npmjs.com/search?q=align+time

## Logged in or logged out?

Doesn't matter.

## Expected behavior

Keywords list is collapsed to some reasonable number of keywords.

## Actual behavior

Whole keywords list is shown:
![](https://user-images.githubusercontent.com/609373/39097520-f1c862e8-4665-11e8-91cd-3a7d05f17487.png)

## Steps to reproduce the problem
1. Go to packages list page with package like [this one](https://www.npmjs.com/package/@nodert-win10-cu/windows.media.core).

**Comment by jefflembeck** [comment]
yeah this is a good idea. I'll put it in our list of todos, thank you.
